packages/@textlint/text-to-ast/src/plaintext-syntax.ts

```diff
@@ -1,11 +1,11 @@
 // LICENSE : MIT
 "use strict";
-const { ASTNodeTypes } = require("@textlint/ast-node-types");
-const exports = {
+import { ASTNodeTypes } from "@textlint/ast-node-types";
+
+export default {
```

Comment:
Can you use named exports as possible?

```suggestion
export const Syntax = {
```

Also need to change to `import { Syntax }`

Reply:
https://basarat.gitbook.io/typescript/main-1/defaultisbad